infrastructure/kubernetes/app-backend.yml

```diff
@@ -51,16 +51,27 @@ spec:
         - name: "AZURE_NH_HUB_NAME"
           value: "agid-notificationhub-test"
         - name: "NODE_ENV"
-          value: "production"
+          # uses redis cluster when this is set to production
+          valueFrom:
+            configMapKeyRef:
+              name: app-backend-config
```

Comment:
we need a yml config for creating this config map (I actually don't see the real need to have a config map here?)

Reply:
The other option is to hardcode the value (ie. "agid-redis-test") and this configuration won't work in production.

This states for other configuration parameters as well (ie. the hardcoded "agid-notificationhub-test" value). I've openened a [pivotal story] https://www.pivotaltracker.com/story/show/158191461) to address the issue.

Other options to make k8 pods work in test *and* in production as well ?)

I guess this values should be taken from the Terraform state someway (ie. terraform show) and set up in the configuration map manually (using kubectl or by producing a yaml file). 

Note that there are already some secrets that are created manually (using kubectl) to make the APP backend work (and they don't appear in the configuration).

Reply:
it's ok for now to hardcode the config, we will have to create yml configs for production (or use helm templates) - we may have terraform create configuration maps in k8s, but I would not try to solve that problem now, unless it's trivial

Reply:
hardcoded